contract_template/cookiecutter.json

```diff
@@ -0,0 +1,9 @@
+{
+    "project_slug": "default_project_slug",
+    "contract_file_name": "default_contract_file_name",
+    "contract_class_name": "default_class_name",
+    "read_functions": {},
+    "write_functions": {},
+    "_jinja2_env_vars": {},
+    "year": "default_year"
+}
```

Comment:
This is a default configuration for cookiecutter that will be overwritten when cookiecutter is called.